guides/transactions.md

```diff
@@ -2,9 +2,9 @@
 
 ## Annotating methods
 
-In DHIS2, transaction boundaries are declared programmatically, via the `@ org.springframework.transaction.annotation.Transactional` annotation on Service methods of concrete classes (as opposed to interfaces).
```

Comment:
Just for clarification: `programmatically` would be if we explicitly invoke some method somewhere in the program.